src/vi/truyenqq/src/eu/kanade/tachiyomi/extension/vi/truyenqq/TruyenQQ.kt

```diff
@@ -133,8 +133,8 @@ class TruyenQQ : ParsedHttpSource() {
 
     // Pages
     override fun pageListParse(document: Document): List<Page> =
-        document.select(".page-chapter img")
-            .mapIndexed { idx, it ->
+        document.select(".page-chapter img:not([src*='stress.gif'])")
+            .mapIndexedNotNull { idx, it ->
```

Comment:
According to [Jsoup docs](https://jsoup.org/apidocs/org/jsoup/nodes/Element.html#select(java.lang.String)) it can't be null.
> **Returns:**
> an [Elements](https://jsoup.org/apidocs/org/jsoup/select/Elements.html) list containing elements that match the query (empty if none match)

```suggestion
            .mapIndexed { idx, it ->
```